.travis.yml

```diff
@@ -45,7 +45,7 @@ matrix:
       if: type = cron
       before_install:
         - python -m pip install cython numpy scipy
-        - python -m pip install --no-build-isolation hypothesis matplotlib pytest pytest-cov pytest-xdist tqdm
+        - python -m pip install --no-build-isolation hypothesis matplotlib pytest pytest-cov pytest-xdist tqdm threadpoolctl
```

Comment:
@yuxuanzhuang if you can add threadpoolctl to either line 27 or 28 that'd be great too (the ppc64le build has it's own install call).